src/util/domain_info_utils.c

```diff
@@ -551,11 +549,8 @@ sss_write_domain_mappings(struct sss_domain_info *domain)
 
     ret = EOK;
 done:
-    err = sss_krb5_touch_config();
-    if (err != EOK) {
-        DEBUG(SSSDBG_CRIT_FAILURE, "Unable to change last modification time "
-              "of krb5.conf. Created mappings may not be loaded.\n");
-        /* Ignore */
+    if (ret == EOK) {
+        sss_krb5_touch_config();
```

Comment:
You are slightly changing the logic, are you sure about this change? I ask this out of ignorance, as I expected a similar logic to the change you made earlier.

Reply:
The purpose of this "touch" is to make apps that monitor 'krb5.conf' reload it.
It only makes sense if SSSD updated anything successfully ("ret == EOK").

Reply:
Ok, thanks for clarifying.